### PR TITLE
Dependency updates

### PR DESCRIPTION
**Pull request by golem131**
Update sbt 0.13.15 -> 0.13.16
Scala 2.12.2 -> 2.12.4
Scala 2.11.11 -> 2.11.12
Scala 2.10.6 -> 2.10.7
Scala-native 0.3.0 -> 0.3.3

Dependency updates:
scalajs 0.6.17 -> 0.6.21

### CODE REVIEW

**Comment by derekmorr** [comment]
This looks ok to me, but I'd like @shadaj to review it. He had found some issues w/ scala native in #282 that he thought would be fixed in 0.3.3.

**Comment by golem131** [comment]
@shadaj pls, review this

**Comment by derekmorr** [comment]
@shadaj Any objections if I merge this?

File: .travis.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ jdk:
 
 matrix:
   include:
-    - scala: 2.12.2
+    - scala: 2.12.3
```
Comment (derekmorr) [inline .travis.yml]: Shouldn't this also be 2.12.4 ?

**Comment by shadaj** [comment]
@derekmorr everything looks good! I ran the unit tests and everything is passing on Scala Native with adjustments for the different implementation of Double.toString.

**Comment by garyKeorkunian** [comment]
Looks like this one has been overridden by a subsequent merge.  It's a bit old and may be irrelevant now.  @golem131 can you verify?

**Comment by garyKeorkunian** [comment]
I am going to close this PR as any version issues can be addressed in the 2.13 update

**Pull request closed without merging**